.pack.mk

```diff
@@ -27,10 +27,34 @@ prepare:
 
 package: prepare
 	if [ "${TARANTOOL_SERIES}" != "1.10" ]; then \
-		if [ -n "${GIT_TAG}" ]; then \
-			export VERSION="$$(echo ${GIT_TAG} | sed 's/-/~/')"; \
+		if [ "${OS}" = "alpine" ]; then \
+			if [ -n "${GIT_TAG}" ]; then \
+				if $$(echo ${GIT_TAG} | grep -q "entrypoint"); then \
```

Comment:
This `if/else` block can be simplified to one line. Isn't it?

Reply:
fixed